charts/tidb-cluster/values.yaml

```diff
@@ -177,8 +180,9 @@ tidb:
       # cloud.google.com/load-balancer-type: Internal
 
 # mysqlClient is used to set password for TiDB
+# it must has Python MySQL client installed
```

Comment:
Having this image be configurable may not be useful anymore. But we could rename it to pythonMysqlClient.

Reply:
It's still need to be configured especially when dockerhub is unavailable.